package.json

```diff
@@ -5,8 +5,9 @@
   "dependencies": {
     "@bugsnag/js": "^7.16.1",
     "@bugsnag/plugin-react": "^7.16.1",
-    "@cosmjs/proto-signing": "^0.27.1",
-    "@cosmjs/stargate": "^0.27.1",
+    "@cosmjs/proto-signing": "^0.28.1",
+    "@cosmjs/stargate": "^0.28.1",
+    "@desmoslabs/desmjs": "^2.0.0-beta.4",
```

Comment:
This is required to get the `profileFromAny` function that converts a Desmos Profile to an Account instance properly